src/extension/features/budget/link-to-inflows/index.js

```diff
@@ -0,0 +1,58 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import {
+  getSelectedMonth,
+  isCurrentRouteBudgetPage,
+  transitionTo,
+} from 'toolkit/extension/utils/ynab';
+
+/**
+ * Adds a click handler to the "TOTAL INFLOW" inspector area.
+ */
+export class LinkToInflows extends Feature {
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage();
+  }
+
+  onRouteChanged() {
+    if (this.shouldInvoke()) {
+      this.invoke();
+    }
+  }
+
+  /**
+   * Add the wrapper element to the TOTAL INFLOWS inspector area.
+   */
+  invoke() {
+    $('.budget-inspector')
+      .find(`h3:contains("${Ember.I18n.t('inspector.totalIncome')}")`)
+      .next()
+      .andSelf()
+      .wrapAll('<span class="toolkit-total-inflows" />');
+
+    $('.toolkit-total-inflows').click(this.onClick);
+  }
+
+  /**
+   * Handle the click on TOTAL INFLOWS. Set the search field and navigate
+   * to the accounts route.
+   */
+  onClick() {
+    const month = getSelectedMonth();
+    const controller = controllerLookup('accounts');
+
+    if (controller.filters) {
+      controller.filters.resetFilters();
+    }
+
+    controller.set(
+      'searchText',
+      `${Ember.I18n.t('search.income')}: ${month.format('MMMM YYYY')}`
+    );
+    transitionTo('accounts');
+  }
+
+  injectCSS() {
```

Comment:
more of a nit but I usually put these class implementation funcs at the top (this one is almost always first) and then all the private functions (`onClick`) after the `invoke`/`observe`